eng/common/scripts/logging.ps1

```diff
@@ -31,10 +31,10 @@ function LogDebug
 {
     if ($isDevOpsRun) 
     {
-        Write-Host "##vso[task.LogIssue type=debug;]$args"
+        Write-Host "[debug]$args"
```

Comment:
Oh on my other comments I assume this was still debug logging. I think I would keep this so we have support for that debug logging. Perhaps we need a LogInfo or something similar that will just do a Write-Host.